examples/typescript/tsconfig.json

```diff
@@ -4,5 +4,6 @@
     "target": "es5",
     "lib": ["dom", "dom.iterable", "esnext"]
   },
-  "include": ["src"]
+  "include": ["src"],
+  "exclude": ["actions", "delivery-functions"]
```

Comment:
Which tsconfig will be used during development? Will we add this to the template?

Reply:
`build-delivery-function.js` has a config :)

Reply:
Happy to see include an extra one if we find settings we see the need